test/development/middleware-errors/index.test.ts

```diff
@@ -55,7 +55,7 @@ describe('middleware - development errors', () => {
       const browser = await next.browser('/')
       await assertHasRedbox(browser)
       await next.patchFile('middleware.js', `export default function () {}`)
-      await assertHasRedbox(browser)
```

Comment:
This looks like they were already wrong before?

Reply:
Yes, and the fact it was passing before was because patchFile did not finish before the assertion